README.md

```diff
@@ -19,6 +19,28 @@ The outputs are used in
 [PyPSA-Eur](https://github.com/PyPSA/pypsa-eur) and
 [PyPSA-Eur-Sec](https://github.com/PyPSA/pypsa-eur-sec).
 
+## Contributions welcome!
+
+We are happy to receive contributions, either as feedback, corrections, modifications or addition of new technology entries.
+You can either open an *Issue* on GitHub for reporting potentials problems, providing feedback or suggesting additional entries.
+Alternatively you can also directly open a *Pull Request* already containing suggested modifications.
+
+All sizes and qualities of contributions are welcome. After an the *Issue* or *Pull Request* we'll assist with comments and leads to get your contribution included into the repository.
+
+## Unit conventions
+
+The following conventions are recommended for new additions to the repository.
+Care must be taken with legacy entries, of which not all follow these conventions.
+
+* Energy units: Use MWh and MW or kWh and kW
```

Comment:
Suggestion: 
-	Investment costs in EUR/kW (or EUR/kWh for stores) -> increase readability
-	Energy and VOM in EUR/MWh